appinventor/appengine/src/com/google/appinventor/client/style/neo/neo.css

```diff
@@ -590,8 +590,9 @@ they get cut off in Windows.
   .ode-Designer-LeftColumn {
     min-width: 265px;
   }
+
   .ode-Designer-RightColumns {
-    min-width: 222px;
+    width: 222px;
```

Comment:
A downside of this change is that if the properties are too long (e.g., in German), then the panel will scroll horizontally rather than becoming wider.

Reply:
> A downside of this change is that if the properties are too long (e.g., in German), then the panel will scroll horizontally rather than becoming wider.

This may be worth going over in person. I set it to fixed width because I was getting bizarre resizing behavior under certain circumstances. I'll see if I can reconstruct it.

Reply:
Given that in the old UI we fixed the width to 222px as well, I think this should be alright for the moment.